cpo_front_end:
  ros__parameters:
    from_serial: false

    data_path: "/home/ben/CLionProjects/gpso/data/rtcm3/feb15c.BIN"

    port_path: "/dev/ttyUSB0"
    baud: 57600

    # Unix time data is from. Set to negative to use live time
    approximate_time: 1613400000       # Feb.15/21